### ISSUE TITLE
Error message colour on (Drupal) Search Results page fails 1.4.3 Contrast (Minimum) Level AA

### ISSUE THREAD

**Issue by mmafe** [ccswbs/hjckrrh#878]
The error message colour contrast on the Search Results page fails 1.4.3 Contrast (Minimum) Level AA.

To recreate:
1. Go to the /search/node page on a site
2. Submit a blank search

**Result:** The words "Please enter some keywords" have a colour of #b94a48. This fails contrast.

Note that we have already fixed the Bootstrap .alert-danger class in base.css. Recommend updating the base.css file so that ```div.error``` and ```table tr.error``` also receive the #b83635 colour that's being applied to ```.alert-danger```.

![image](https://user-images.githubusercontent.com/1927902/47572216-745cba80-d908-11e8-8415-b5970b06f86e.png)
